layout/_third-party/comments/gitalk.swig

```diff
@@ -0,0 +1,24 @@
+{% if page.comments && theme.gitalk.enable %}
+
+  <link rel="stylesheet" href="https://unpkg.com/gitalk/dist/gitalk.css">
+
+  <script src="https://unpkg.com/gitalk/dist/gitalk.min.js"></script>
+
+   <script type="text/javascript">
+        var gitalk = new Gitalk({
+          clientID: '{{ theme.gitalk.ClientID }}',
+          clientSecret: '{{ theme.gitalk.ClientSecret }}',
+          repo: '{{ theme.gitalk.repo }}',
+          owner: '{{ theme.gitalk.owner }}',
+          admin: ['{{ theme.gitalk.adminUser }}'],
+          id: {{ theme.gitalk.ID }},
+          labels: '{{ theme.gitalk.labels }}'
```

Comment:
Missing ',' at the end of line 15.

Reply:
`labels: ['{{ theme.gitalk.labels }}'],`
Labels should be an array, or will get concat error.